lib/powerpoint.js

```diff
@@ -0,0 +1,79 @@
+var app = require('app')
+var BrowserWindow = require('browser-window')
+var officegen = require('officegen')
+var pptx = officegen('pptx')
+var fs = require('fs')
+var ipc = require('ipc')
+
+module.exports = () => {
+    var out = fs.createWriteStream(app.windows[0].getTitle() + '.pptx')
+    var deckLength = 0
+    var win = app.windows[0]
+    var pdfWin = new BrowserWindow({
+        toolbar: false,
+        center: true,
+        width: 987,
+        height: 680,
+        show: false
+    })
+
+    pdfWin.loadUrl('http://localhost:2000/')
+    setTimeout(function () {
+        pdfWin.webContents.executeJavaScript(`
+            var ipc = require('ipc')
+            ipc.on('asynchronous-message', (event, arg) => {
```

Comment:
you don't need specify asynchronous, that's known - just `message` and `reply` events would be fine